### ISSUE TITLE
Not detecting members of scipy.special

### ISSUE THREAD

**Issue by fjhheras** [PyCQA/pylint#3703]
expit is not recognised as a member of scipy.special

### Steps to reproduce
Run pylint on this file:
```
import scipy.special
print(scipy.special.expit(10))
```
### Current behavior
pylint throws `E1101: Module 'scipy.special' has no 'expit' member (no-member)` and gives my code an unfair low rating of -10.00/10

### Expected behavior
pylint does not throw that error, because expit is a member of scipy.special

### pylint --version output
pylint 2.5.2
astroid 2.4.1
Python 3.8.3 (default, May 19 2020, 18:47:26) 
[GCC 7.3.0]

and with:

pylint 2.5.3
astroid 2.4.2
Python 3.8.3 (default, May 19 2020, 18:47:26) 
[GCC 7.3.0]

**Comment by PCManticore** [comment]
`scipy.special` is a C library, which pylint cannot understand as it requires Python source code. You can try running `pylint` with `--extension-pkg-whitelist=scipy.spatial` and see if that works or you can ignore the module entirely with `--ignored-modules=scipy.spatial`

https://github.com/scipy/scipy/tree/master/scipy/special
